### PR TITLE
Add `Slider` component

### LINKED ISSUE #92: Slider Component
image

### PR DESCRIPTION
**Pull request by hrach**
RangeSlider will be added after updating to M3-1.1 version.

closes #92

### CODE REVIEW

File: ui/src/main/java/kiwi/orbit/compose/ui/controls/Slider.kt [hunk 1]
```diff
@@ -0,0 +1,362 @@
...
+    }.progressSemantics(value, valueRange, steps)
+}
+
+private val TrackSize = 4.dp
+private val TickSize = 2.dp
+
+@OrbitPreviews
+@Composable
+internal fun SliderPreview() {
+    Preview {
```
Comment (gerak-cz) [inline ui/src/main/java/kiwi/orbit/compose/ui/controls/Slider.kt:328]: The previews are really confusing. Can we put perhaps a "gray background" behind them so they can be easily separated? Similar to Card.

**Review by gerak-cz — APPROVED** [review]

File: docs/03-components/02-structure/card.mdx [hunk 2]
```diff
@@ -4,17 +4,17 @@ title: Android
 
 ## Overview
 
-`Card` is a rectangular surface that separates content into sections.
+`Card` is a provided for content separation into sections.
```
[the remark this answers is not in the document]
Comment (hrach) [reply docs/03-components/02-structure/card.mdx:7]: I have returned it but keep in mind that this documentation is not to explain what it is, there is another tab for it, this page is how to use/what to use.

File: docs/03-components/02-structure/card.mdx [hunk 3]
```diff
@@ -4,17 +4,17 @@ title: Android
...
 
 - [`Card`](https://kiwicom.github.io/orbit-compose/ui/kiwi.orbit.compose.ui.controls/-card.html)
 
-For rounded and elevated card see
+For rounded and elevated Material-like card see
```
[the remark this answers is not in the document]
Comment (hrach) [reply docs/03-components/02-structure/card.mdx:11]: AFAIK they are not [coordinated adjectives](https://prowritingaid.com/grammar/1008103/Should-I-use-a-comma-between-two-adjectives) as you wouldn't say "elevated and Material-like card".

File: ui/src/main/java/kiwi/orbit/compose/ui/controls/Slider.kt (hunk 1 above)
Comment (hrach) [reply ui/src/main/java/kiwi/orbit/compose/ui/controls/Slider.kt:328]: added separators

File: docs/03-components/02-structure/card.mdx (hunk 3 above)
[the remark this answers is not in the document]
Comment (gerak-cz) [reply docs/03-components/02-structure/card.mdx:11]: I think that without the comma it is not clear whether the Material-like card is rounded and elevated or if the card is rounded and elevated and also Material-like.

**Pull request merged**